src/psyclone/tests/psyGen_test.py

```diff
@@ -2445,6 +2445,9 @@ def test_node_abstract_methods():
     with pytest.raises(NotImplementedError) as err:
         Node.gen_code(loop)
     assert ("Please implement me" in str(err))
+    with pytest.raises(NotImplementedError) as err:
+        Node.gen_c_code(loop)
+    assert ("Please implement me" in str(err))
```

Comment:
pylint complains that the parentheses are unnecessary.

Reply:
Fixed